dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp

```diff
@@ -321,9 +321,20 @@
                         contentlet, LegacyFieldTransformer.from(field),
                         user);
                 defaultPathFolderPathIds.add(0, "root");
+                
+                boolean dragAndDrop = true;
+
+                List<FieldVariable> fieldVariables=APILocator.getFieldAPI().getFieldVariablesForField(field.getInode(), user, true);
+                    for(FieldVariable fv : fieldVariables){
+                        if (fv.getKey().equals("drag-and-drop")) {
+                            dragAndDrop = fv.getValue().toUpperCase().equals("FALSE") ? false : true;
```

Comment:
can be simplified to 
dragAndDrop = fv.getValue().equalsIgnoreCase("true");
But testing null is not bad either.